{"uid":"3a34f79bedc98200","name":"test_database_relation","fullName":"tests.integration.test_charm#test_database_relation","historyId":"a46dd5fec5170ae1b5112e2bb246aae9","time":{"start":1739237896668,"stop":1739238916753,"duration":1020085},"description":"Test the database relation.","descriptionHtml":"<p>Test the database relation.</p>\n","status":"broken","statusMessage":"TimeoutError: Timed out waiting for model:\nmysql-k8s/0 [executing] active: \n  mysql-k8s/1 [executing] active: \n  mysql-k8s/2 [executing] active: Primary","statusTrace":"Traceback (most recent call last):\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/runner.py\", line 341, in from_call\n    result: Optional[TResult] = func()\n                                ^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/runner.py\", line 241, in <lambda>\n    lambda: runtest_hook(item=item, **kwds), when=when, reraise=reraise\n            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n           ^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/threadexception.py\", line 87, in pytest_runtest_call\n    yield from thread_exception_runtest_hook()\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/threadexception.py\", line 63, in thread_exception_runtest_hook\n    yield\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/unraisableexception.py\", line 90, in pytest_runtest_call\n    yield from unraisable_exception_runtest_hook()\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/unraisableexception.py\", line 65, in unraisable_exception_runtest_hook\n    yield\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/logging.py\", line 850, in pytest_runtest_call\n    yield from self._runtest_for(item, \"call\")\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/logging.py\", line 833, in _runtest_for\n    yield\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/capture.py\", line 878, in pytest_runtest_call\n    return (yield)\n            ^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/skipping.py\", line 257, in pytest_runtest_call\n    return (yield)\n            ^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/runner.py\", line 183, in pytest_runtest_call\n    raise e\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/runner.py\", line 173, in pytest_runtest_call\n    item.runtest()\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/python.py\", line 1632, in runtest\n    self.ihook.pytest_pyfunc_call(pyfuncitem=self)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n           ^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/python.py\", line 162, in pytest_pyfunc_call\n    result = testfunction(**testargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pytest_asyncio/plugin.py\", line 529, in inner\n    _loop.run_until_complete(task)\n  File \"/usr/lib/python3.12/asyncio/base_events.py\", line 687, in run_until_complete\n    return future.result()\n           ^^^^^^^^^^^^^^^\n  File \"/root/spread_project/tests/integration/test_charm.py\", line 96, in test_database_relation\n    await ops_test.model.wait_for_idle(\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/juju/model.py\", line 3209, in wait_for_idle\n    raise jasyncio.TimeoutError(\"Timed out waiting for model:\\n\" + busy)\nTimeoutError: Timed out waiting for model:\nmysql-k8s/0 [executing] active: \n  mysql-k8s/1 [executing] active: \n  mysql-k8s/2 [executing] active: Primary","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":1,"retriesStatusChange":true,"beforeStages":[{"name":"event_loop","time":{"start":1739237896470,"stop":1739237896471,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"ops_test","time":{"start":1739237896471,"stop":1739237896668,"duration":197},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"tmp_path_factory","time":{"start":1739237896471,"stop":1739237896471,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"abort_on_fail","time":{"start":1739237896668,"stop":1739237896668,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"charm","time":{"start":1739237896668,"stop":1739237896668,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"testStage":{"description":"Test the database relation.","status":"broken","statusMessage":"TimeoutError: Timed out waiting for model:\nmysql-k8s/0 [executing] active: \n  mysql-k8s/1 [executing] active: \n  mysql-k8s/2 [executing] active: Primary","statusTrace":"Traceback (most recent call last):\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/runner.py\", line 341, in from_call\n    result: Optional[TResult] = func()\n                                ^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/runner.py\", line 241, in <lambda>\n    lambda: runtest_hook(item=item, **kwds), when=when, reraise=reraise\n            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n           ^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/threadexception.py\", line 87, in pytest_runtest_call\n    yield from thread_exception_runtest_hook()\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/threadexception.py\", line 63, in thread_exception_runtest_hook\n    yield\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/unraisableexception.py\", line 90, in pytest_runtest_call\n    yield from unraisable_exception_runtest_hook()\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/unraisableexception.py\", line 65, in unraisable_exception_runtest_hook\n    yield\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/logging.py\", line 850, in pytest_runtest_call\n    yield from self._runtest_for(item, \"call\")\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/logging.py\", line 833, in _runtest_for\n    yield\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/capture.py\", line 878, in pytest_runtest_call\n    return (yield)\n            ^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/skipping.py\", line 257, in pytest_runtest_call\n    return (yield)\n            ^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/runner.py\", line 183, in pytest_runtest_call\n    raise e\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/runner.py\", line 173, in pytest_runtest_call\n    item.runtest()\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/python.py\", line 1632, in runtest\n    self.ihook.pytest_pyfunc_call(pyfuncitem=self)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n           ^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/python.py\", line 162, in pytest_pyfunc_call\n    result = testfunction(**testargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pytest_asyncio/plugin.py\", line 529, in inner\n    _loop.run_until_complete(task)\n  File \"/usr/lib/python3.12/asyncio/base_events.py\", line 687, in run_until_complete\n    return future.result()\n           ^^^^^^^^^^^^^^^\n  File \"/root/spread_project/tests/integration/test_charm.py\", line 96, in test_database_relation\n    await ops_test.model.wait_for_idle(\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/juju/model.py\", line 3209, in wait_for_idle\n    raise jasyncio.TimeoutError(\"Timed out waiting for model:\\n\" + busy)\nTimeoutError: Timed out waiting for model:\nmysql-k8s/0 [executing] active: \n  mysql-k8s/1 [executing] active: \n  mysql-k8s/2 [executing] active: Primary","steps":[],"attachments":[{"uid":"ab09e8f68279207c","name":"log","source":"ab09e8f68279207c.txt","type":"text/plain","size":7550}],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":1,"hasContent":true,"attachmentStep":false},"afterStages":[{"name":"abort_on_fail::0","time":{"start":1739238916798,"stop":1739238916798,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"ops_test::finalizer","time":{"start":1739238916799,"stop":1739238917545,"duration":746},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::2","time":{"start":1739238917545,"stop":1739238917545,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_close_event_loop","time":{"start":1739238917546,"stop":1739238917546,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_provide_clean_event_loop","time":{"start":1739238917546,"stop":1739238917546,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"tag","value":"abort_on_fail"},{"name":"tag","value":"asyncio"},{"name":"parentSuite","value":"tests.integration"},{"name":"suite","value":"test_charm"},{"name":"host","value":"fv-az1374-838"},{"name":"thread","value":"19949-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.integration.test_charm"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[{"uid":"d37fff34f555f03a","status":"unknown","time":{}}],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":0,"broken":53,"skipped":0,"passed":179,"unknown":10,"total":242},"items":[{"uid":"91f675e785967b49","reportUrl":"../2529//#testresult/91f675e785967b49","status":"broken","statusDetails":"TimeoutError: Timed out waiting for model:\nmysql-k8s/0 [executing] active: \n  mysql-k8s/1 [executing] active: \n  mysql-k8s/2 [executing] active: Primary","time":{"start":1739151501151,"stop":1739152520836,"duration":1019685}},{"uid":"e55fa0b6a9ca781b","reportUrl":"../2528//#testresult/e55fa0b6a9ca781b","status":"passed","time":{"start":1739065146745,"stop":1739065653003,"duration":506258}},{"uid":"d13b88438140415b","reportUrl":"../2527//#testresult/d13b88438140415b","status":"passed","time":{"start":1738978644406,"stop":1738979208991,"duration":564585}},{"uid":"98286b52d645f5b2","reportUrl":"../2524//#testresult/98286b52d645f5b2","status":"broken","statusDetails":"TimeoutError: Timed out waiting for model:\nmysql-k8s/0 [executing] active: Primary\n  mysql-k8s/1 [executing] active: \n  mysql-k8s/2 [executing] active:","time":{"start":1738892249084,"stop":1738893266449,"duration":1017365}},{"uid":"cbae32d10e87486c","reportUrl":"../2520//#testresult/cbae32d10e87486c","status":"broken","statusDetails":"TimeoutError: Timed out waiting for model:\nmysql-k8s/0 [executing] active: \n  mysql-k8s/1 [executing] active: \n  mysql-k8s/2 [executing] active: Primary","time":{"start":1738805998201,"stop":1738807028748,"duration":1030547}},{"uid":"cab172a181535698","reportUrl":"../2518//#testresult/cab172a181535698","status":"passed","time":{"start":1738719477847,"stop":1738719951325,"duration":473478}},{"uid":"b4bd495f9883e53e","reportUrl":"../2514//#testresult/b4bd495f9883e53e","status":"broken","statusDetails":"TimeoutError: Timed out waiting for model:\nmysql-k8s/0 [executing] active: \n  mysql-k8s/1 [executing] active: Primary\n  mysql-k8s/2 [executing] active:","time":{"start":1738633034372,"stop":1738634050901,"duration":1016529}},{"uid":"f86cbd30e954084d","reportUrl":"../2510//#testresult/f86cbd30e954084d","status":"passed","time":{"start":1738546645557,"stop":1738547357525,"duration":711968}},{"uid":"bd6c928b4f6f79c9","reportUrl":"../2509//#testresult/bd6c928b4f6f79c9","status":"broken","statusDetails":"TimeoutError: Timed out waiting for model:\nmysql-k8s/0 [executing] active: \n  mysql-k8s/1 [executing] active: Primary\n  mysql-k8s/2 [executing] active:","time":{"start":1738460241956,"stop":1738461256800,"duration":1014844}},{"uid":"42bbe574b20671e3","reportUrl":"../2508//#testresult/42bbe574b20671e3","status":"passed","time":{"start":1738373848027,"stop":1738374111192,"duration":263165}},{"uid":"4e2978c226219cdd","reportUrl":"../2493//#testresult/4e2978c226219cdd","status":"broken","statusDetails":"TimeoutError: Timed out waiting for model:\nmysql-k8s/0 [executing] active: \n  mysql-k8s/1 [executing] active: \n  mysql-k8s/2 [executing] active: Primary","time":{"start":1738287445053,"stop":1738288467314,"duration":1022261}},{"uid":"4e43e3b7dea442c9","reportUrl":"../2464//#testresult/4e43e3b7dea442c9","status":"broken","statusDetails":"TimeoutError: Timed out waiting for model:\nmysql-k8s/0 [executing] active: \n  mysql-k8s/1 [executing] active: Primary\n  mysql-k8s/2 [executing] active:","time":{"start":1738201066038,"stop":1738202086847,"duration":1020809}},{"uid":"f3752811f721849d","reportUrl":"../2444//#testresult/f3752811f721849d","status":"passed","time":{"start":1738114715412,"stop":1738115067472,"duration":352060}},{"uid":"6629e702069252a6","reportUrl":"../2431//#testresult/6629e702069252a6","status":"broken","statusDetails":"TimeoutError: Timed out waiting for model:\nmysql-k8s/0 [executing] active: \n  mysql-k8s/1 [executing] active: Primary\n  mysql-k8s/2 [executing] active:","time":{"start":1738028248166,"stop":1738029262029,"duration":1013863}},{"uid":"51f2701aa0decb84","reportUrl":"../2422//#testresult/51f2701aa0decb84","status":"passed","time":{"start":1737942023978,"stop":1737942389721,"duration":365743}},{"uid":"33757977d7221026","reportUrl":"../2421//#testresult/33757977d7221026","status":"passed","time":{"start":1737855564861,"stop":1737855846639,"duration":281778}},{"uid":"534180bd2a8e214b","reportUrl":"../2420//#testresult/534180bd2a8e214b","status":"broken","statusDetails":"TimeoutError: Timed out waiting for model:\nmysql-k8s/0 [executing] active: \n  mysql-k8s/1 [executing] active: Primary\n  mysql-k8s/2 [executing] active:","time":{"start":1737769057635,"stop":1737770085775,"duration":1028140}},{"uid":"e6a48a3c5cbe660d","reportUrl":"../2373//#testresult/e6a48a3c5cbe660d","status":"passed","time":{"start":1737682666020,"stop":1737682942451,"duration":276431}},{"uid":"f0bc13b4dc34f903","reportUrl":"../2332//#testresult/f0bc13b4dc34f903","status":"passed","time":{"start":1737596259309,"stop":1737596860140,"duration":600831}},{"uid":"c18ec6655d4ded2","reportUrl":"../2269//#testresult/c18ec6655d4ded2","status":"passed","time":{"start":1737510127359,"stop":1737510701112,"duration":573753}}]},"tags":["asyncio","abort_on_fail"]},"source":"3a34f79bedc98200.json","parameterValues":[]}